templates/managed/index.html

```diff
@@ -1,450 +1,592 @@
+      </p>
+    </div>
+  </div>
+
+  <div class="row u-vertically-spaced">
+    <div class="col-4 u-sv3">
+      <h3 class="p-heading--four">Business-centric IT</h3>
+      <p>You drive your business, we manage your cloud infrastructure and applications. Expand globally without hiring a global team.</p>
+    </div>
+
+    <div class="col-4 u-sv3">
+      <h3 class="p-heading--four">Reliable and robust</h3>
+      <p>Guaranteed 99.9% production-grade SLAs and best practice solution design. Have your cloud managed by technology-specific experts.</p>
+    </div>
+
+    <div class="col-4 u-sv3">
+      <h3 class="p-heading--four">Flexible pricing</h3>
+      <p>Transparent pricing to suit your needs and help you grow your cloud with predictable costs.</p>
+    </div>
+
+    <div class="col-4 u-sv3">
+      <h3 class="p-heading--four">Security first</h3>
+      <p>Industry wide security best practices and compliance certifications to keep your cloud secure at all times.</p>
+    </div>
+
+    <div class="col-4 u-sv3">
+      <h3 class="p-heading--four">Simple open source</h3>
+      <p>Cut down your operations costs by up to 50% and consume open source without complexity.</p>
+    </div>
+
+    <div class="col-4 u-sv3">
+      <h3 class="p-heading--four">No lock-in</h3>
+      <p>Build a flexible and agnostic cloud with no lock-in and leverage a simplified open source approach. We give you the keys when you are ready to drive.</p>
+    </div>
+  </div>
+  <div class="u-fixed-width">
+    <p>
+      <a href="/managed/contact-us" class="p-button--positive js-invoke-modal">Contact us</a>
+    </p>
+  </div>
+</section>
+
+<hr>
 
+
+<section class=" p-strip">
+  <div class="row">
+    <div class="u-fixed-width">
+      <div class="row">
         <div class="col-4 u-sv3">
-            <h3 class="p-heading--four">Reliable and robust</h3>
-            <p>Guaranteed 99.9% production-grade SLAs and best practice solution design. Have your cloud managed by technology specific experts.</p>
+          {{
+          image(
+          url="https://assets.ubuntu.com/v1/dcb2963c-openstack+cloud+outlines.svg",
+          alt="Openstack cloud outlines",
+          width="263",
+          height="150",
+          hi_def=True,
+          loading="lazy",
+          attrs={"class":"u-hide--small"},
+          ) | safe
+          }}
+          <h3 class="p-heading--four"><a href="/openstack/managed">Openstack</a></h3>
+          <p>Cut your cloud operational costs by up to 50% and build a business centric IT department focused on driving your business. </p>
+          <p><a href="/openstack/managed">Managed OpenStack&nbsp;&rsaquo;</a></p>
```

Comment:
In the doc these links are button and the title 'Learn more about manager IT services' is above this section